CHANGELOG.md

```diff
@@ -129,6 +129,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add app support for events - #7622 by @IKarbowiak
 - Add date & date time attributes - #7500 by @piotrgrundas
 - Add `withChoices` flag for Attribute type - #7733 by @dexon44
+- Add ExternalNotificationTrigger mutation - #7821 by @mstrumeck
```

Comment:
Please rebase the PR with master and move this changelog entry to the "3.1" section - these changes will be released in 3.1.